bom/application/pom.xml

```diff
@@ -54,8 +54,8 @@
         <microprofile-jwt.version>2.1</microprofile-jwt.version>
         <microprofile-lra.version>2.0</microprofile-lra.version>
         <microprofile-openapi.version>3.1</microprofile-openapi.version>
-        <smallrye-common.version>2.1.0</smallrye-common.version>
-        <smallrye-config.version>3.1.3</smallrye-config.version>
+        <smallrye-common.version>2.0.0</smallrye-common.version>
```

Comment:
Is it normal you downgrade smallrye-common?

Reply:
No. Probably a messed rebase.